app/Console/Commands/NotifyVerificationCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Console\Commands;
+
+
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+class NotifyVerificationCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'users:notify-verification {email}';
```

Comment:
аргумент не нужен, нужно пройтись по всем пользователям
```suggestion
    protected $signature = 'users:notify-verification';
```